tutor/src/models/course.ts

```diff
@@ -320,7 +324,7 @@ export class Course extends BaseModel {
     async save() {
         const data = await this.api.request<CourseData>(
             this.isNew ? urlFor('createCourse') : urlFor('updateCourse', { courseId: this.id }),
-            { data: pick(this, SAVEABLE_ATTRS) },
+            { data: pick(this, this.saveAttrs) },
```

Comment:
Since the method's only called from one spot, how about making the method `saveData` and have it do the `pick`